README.md

```diff
@@ -7,7 +7,7 @@ Try .NET is an inline code runner and playground that enables people to experime
 **Note**: Currently,Try .NET only supports C# but we do plan to enable support F# and VB in the near future. 
 
 ### What is the repo for?
-- A place where you can post issues about the [Try .NET](https://docs.microsoft.com/en-us/dotnet/api/system.datetime?view=netframework-4.7.2#invoke-constructors).
+- A place where you can post issues about the [Try .NET](https://try.dot.net/).
```

Comment:
Thanks for noticing this. 

The URL should probably be https://dotnet.microsoft.com/platform/try-dotnet.

Also, would you mind removing the word "the"?

Reply:
No worries, I have made the requested changes!

Reply:
Thanks!